### ISSUE TITLE
Datepicker always have to be in open state

### ISSUE THREAD

**Issue by Grinv** [Hacker0x01/react-datepicker#1179]
I need datepicker input and popper, but popper always has to be in open state. How to do it?
Datepicker is hidden when I click outside.
`inline` prop is not suitable for me because I still need an input too.

**Comment by alex-shamshurin** [comment]
Could you explain  your case in detail? I don't really get what you meant.

**Comment by Grinv** [comment]
@alex-shamshurin of course.
I need the ability to edit the date from two places: input and calendar. But calendar hasn't to be popper. it always be visible. I changed styles for it:
```css
.react-datepicker-popper {
   position: relative !important;
   transform: inherit !important;
}
```
When I call component with `startOpen` prop. it's helps for first run, but when I click outside, calendar hides. I don't know how to cancel this behavior. Calendar always has to be visible,
P.S. I attached the screenshot.

![screenshot](https://user-images.githubusercontent.com/4070730/34114273-66759f0c-e423-11e7-96c0-d7bbf606d298.PNG)
